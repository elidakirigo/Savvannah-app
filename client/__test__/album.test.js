import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'
import Page from '../src/app/album/[id]/page'

describe('render the album page', () => {
	it('renders albumpage unchanged', () => {
		const { container } = render(<Page />)
		expect(container).toMatchSnapshot()
	})
})
