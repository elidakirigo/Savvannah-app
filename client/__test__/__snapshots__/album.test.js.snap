// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the album page should render Albumpage successfully 1`] = `
<div>
  <div
    class="relative overflow-x-auto shadow-lg sm:rounded-lg m-5 max-w-[900px] border-[2px]"
  >
    <h1
      class="text-[30px] p-5 text-purple-700 text-bold"
      data-testid="albumId-header"
    >
      AlbumID : 
      1
    </h1>
    <table
      class="w-full text-sm text-left rtl:text-right text-gray-500 dark:text-gray-400 "
    >
      <thead
        class="text-xs text-gray-700 uppercase dark:text-gray-400"
      >
        <tr>
          <th
            class="px-6 py-3 bg-gray-50 dark:bg-gray-800"
            data-testid="imgtittle-header"
            scope="col"
          >
            image title
          </th>
          <th
            class="px-6 py-3"
            data-testid="image-header"
            scope="col"
          >
            image
          </th>
          <th
            class="px-6 py-3 bg-gray-50 dark:bg-gray-800"
            data-testid="imgid-header"
            scope="col"
          >
            image id
          </th>
          <th
            class="px-6 py-3"
            data-testid="albumId-head"
            scope="col"
          >
            album id
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          class="border-b border-gray-200 dark:border-gray-700 cursor-pointer"
          data-testid="albumLink"
        >
          <th
            class="px-6 py-4 font-medium text-gray-900 max-w-36 bg-gray-50 dark:text-white dark:bg-gray-800 hover:bg-slate-500"
            data-testid="albumtitle"
            scope="row"
          >
            accusamus beatae ad facilis cum similique qui sunt
          </th>
          <td
            class="px-6 py-4 hover:bg-slate-600"
          >
            <img
              alt="accusamus beatae ad facilis cum similique qui sunt"
              data-nimg="1"
              data-testid="image"
              decoding="async"
              height="50"
              loading="lazy"
              src="https://via.placeholder.com/600/92c952"
              style="color: transparent;"
              width="50"
            />
          </td>
          <td
            class="px-6 py-4 bg-gray-50 dark:bg-gray-800"
            data-testid="imageId"
          >
            1
          </td>
          <td
            class="px-6 py-4"
            data-testid="albumId"
          >
            1
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
